src/components/molecules/ToDoItem.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Span } from "../atoms/Span";
+import { styled } from "styled-components";
+import { ImageButton } from "../atoms/ImageButton";
+import deleteButton from "../../image/delete.png";
+import backButton from "../../image/restore.png";
+import { useRecoilState } from "recoil";
+import { nowTodoList, solvedTodoList } from "../../recoil/atom.ts";
+function ToDoItem({ todo, isSolved }) {
+  const [nowTodo, setNowTodo] = useRecoilState(nowTodoList);
+  const [solvedTodo, setSolvedTodo] = useRecoilState(solvedTodoList);
+  const deleteButtonClick = () => {
+    const newSolvedList = solvedTodo.filter((item) => item !== todo);
+    setSolvedTodo(newSolvedList);
+  };
+  const backButtonClick = () => {
+    const newSolvedList = solvedTodo.filter((item) => item !== todo);
+    setSolvedTodo(newSolvedList);
+    setNowTodo([...nowTodo, todo]);
+  };
+  const clickNowToDo = (e) => {
+    // SolvedToDo 아이템 클릭했을 경우, 아무 일도 발생하지 않기하기
+    if (isSolved) {
+      return;
+    } else {
+      const targetTodo = e.target.innerText;
+      const newNowTodoList = nowTodo.filter((item) => item !== targetTodo);
+      setNowTodo(newNowTodoList);
+      setSolvedTodo([...solvedTodo, targetTodo]);
+    }
```

Comment:
else문은 굳이 필요 없을 것 같아요!
```suggestion
    if (isSolved) {
      return;
    }
    const targetTodo = e.target.innerText;
    const newNowTodoList = nowTodo.filter((item) => item !== targetTodo);
    setNowTodo(newNowTodoList);
    setSolvedTodo([...solvedTodo, targetTodo]);
```